### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by kaefe64** [kaefe64/Arduino_uSDX_Pico_FFT_Proj#9] [Q&A]
If you want to say something...

**Comment by R4HFW** [comment]
В последней прошивке не работает сохранение настроек. Вместо этого включается микрофон и идет фон, после сброса питания пропадает.

**Reply by kaefe64** [reply]
Hi Aleksej R4HFW

Translating your msg I got this:
In the last firmware, the preservation of settings does not work. Instead, the microphone is turned on and the background goes, after a power supply disappears.

Thank you for your feedback.
Could you give more information about the issue? PCB used (original or other?), Arduino IDE version, Library version.
With "preservation of settings does not work", do you mean the Memory Save command is now working?
What do you mean with "microphone is turned on"? The PTT is ON and starts transmitting?

I haven't tested all features on every new firmware, so some mistakes may occur, sorry.

73
Klaus F
PY2KLA

**Reply by R4HFW** [reply]
Hello.
Arjan v2 board with all your modifications. Use Arduino IDE version 2.0.1 in Linux/Ubuntu.
I use the TFT_eSPI library from Bodmer. Arduino Mbed OS RP2040 board — 4.0.2.
I set the frequency, the modulation type on all ranges, select save in the menu and then the microphone turns on and there is background, the transmission does not turn on. After turning the transceiver off and on, the background disappears, but the settings are not saved. Moreover, the audio is saved and read normally. Everything else works fine. The project is very good. Thank you for the work done. 
73
Aleksej R4HFW

**Comment by R4HFW** [comment]
Здравствуйте.
Плата Arjan v2 со всеми вашими доработками. используйте Arduino IDE версии 2.0.1 в Linux/Ubuntu.
Использую библиотеку TFT_eSPI от Bodmer. Плата Arduino Mbed OS RP2040 — 4.0.2.
Настраиваю каналы, вид модуляции на всех соединениях, выбираю в меню сохранение и тут включается микрофон и идет фон. После выключения и включения трансивера фон пропадает, но настройки не отключаются. Причём аудио сохраняется и читается нормально. Остальное всё работает хорошо. Проект очень хороший. Спасибо Вам за проделанную работу
73
Алексей R4HFW

**Reply by kaefe64** [reply]
"Hello.
Arjan v2 board with all your improvements. use the Arduino ID version 2.0.1 in Linux/Ubuntu.
I use the TFT_eSPI library from Bodmer. Arduino Mbed OS RP2040 – 4.0.2.
I adjust the channels, the kind of modulation on all connections, I choose the preservation in the menu and here the microphone is turned on and the background goes. After turning off and switching on, the background disappears, but the settings are not turned off. And the audio is stored and read normally. The rest is working well. The project is very good. Thank you for the work done
73
Alexey R4HFW"



Hi Alexey R4HFW

Thank you for the information.
I tested on my Arjan and I can see that something is wrong when I try to "Save", but it is not exact as your description.
I will need to check this issue. Thanks for telling me.

Are you using the Arduino Pro Mini for the relays? or the PCF8574's?
If everybody uses the Arduino Pro Mini, I could change the routines to save data, that are complex using the Pico's Flash Memory, and use instead the Pro Mini **EEProm** via I2C.

73
Klaus F
PY2KLA

**Reply by R4HFW** [reply]
Hello Klaus.
For relay switching I use PCF8574, and for swr arduino pro mini.
73
Aleksej R4HFW

**Comment by R4HFW** [comment]
![20250625_205414](https://github.com/user-attachments/assets/ac6f4e9c-2f0b-43ea-908c-2e8e3c4bd69f)
![20250625_20
![20250625_205058](https://g
![20250625_205024](https://github.com/user-attachments/assets/9d3c205a-fc47-4f97-95c1-e6843ac01479)
ithub.com/user-attachments/assets/b9e14399-e757-4e80-ac53-cbd0b3e3850a)
![20250625_205047](https://github.com/user-attachments/assets/c51c755e-4840-4bbd-8015-bbbe18bbb015)
5238](https://github.com/user-attachments/assets/2d446d03-c626-4fa9-bf34-3d83004f4c4a)
![20250625_205029](https://github.com/user-attachments/assets/2a87a788-4dd2-4657-ac1c-c83d6a62ea67)

**Comment by R4HFW** [comment]
![20250625_205047](https://github.com/user-attachments/assets/98531b5d-a62a-4b5a-abf0-cb1f89755cbc)
![20250625_205047](https://github.com/user-attachments/assets/fa439fc2-90a2-4909-8332-b7084abc3072)

**Comment by R4HFW** [comment]
![20250625_205024](https://github.com/user-attachments/assets/bc0c0f7c-ee12-4806-b66a-d16e7a2067cd)
![20250625_205047](https://github.com/user-attachments/assets/b9fd2f0c-57fe-45d7-80b9-d6f551a737fb)

**Comment by kaefe64** [comment]
Hi Aleksej R4HFW

Great pictures, thank you for sharing with us.
As I see, you used the PCB like Arjan uSDR Pico (now, he included a TFT display too, but used different pins).
The project looks simple, but it takes a lot of effort with so many details. Great job.

My work is taking some time, but as soon as I have free time, I will work on the issue you reported ("Save" command not working).

Tks, 73
Klaus F
PY2KLA

**Comment by R4HFW** [comment]
Hello Klaus.
Yes, I made the Arjan uSDR Pico boards myself, then your project caught my attention and I decided not to remake it, but to refine it. The problem with the (save) command doesn't bother me much, everything else works.
Thank you. We'll wait for updates.
73
Aleksej R4HFW
